code/src/Form.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react'
+import { Dropdown } from 'Dropdown'
+import { Radio } from 'Radio'
+import { Text } from 'Text'
+import { Submit } from 'Submit'
+
+
+
```

Comment:
same as above and no need for so many empty lines 🙈

Reply:
fixed